src/main/java/de/aservo/atlassian/confapi/model/LicenseBean.java

```diff
@@ -0,0 +1,58 @@
+package de.aservo.atlassian.confapi.model;
+
+import com.atlassian.sal.api.license.SingleProductLicenseDetailsView;
+import de.aservo.atlassian.confapi.constants.ConfAPI;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import java.util.Collection;
+import java.util.Date;
+
+/**
+ * Bean for licence infos in REST requests.
+ */
+@Data
+@NoArgsConstructor
+@XmlRootElement(name = ConfAPI.LICENSE)
+public class LicenseBean {
+
+    @XmlElement
+    private String productName;
+
+    @XmlElement
+    private String licenseType;
+
+    @XmlElement
+    private String organization;
+
+    @XmlElement
+    private String description;
+
+    @XmlElement
+    private Date expiryDate;
+
+    @XmlElement
+    private int numUsers;
+
+    @XmlElement
+    private String key;
+
+    @XmlElement
+    private Collection<String> applicationKeys;
```

Comment:
Ich vermute, productName und applicationKey ist das gleiche, für Jira brauchen wir aber die Collection. Ich finde applicationKeys etwas griffiger, vor allem bei Jira, wo es ja in erster Linie gebraucht wird.

Reply:
Kommando zurück: key und key ist nicht gut. Oder einfach:

private Collection<String> products;

Reply:
Oder applications, bin mir hier unschlüssig...

Reply:
key und applicationKeys kommen von dir, aus dem jira Projekt ;-) Wollte da nicht zwischen funken. Aber gut, habe mal in products umbenannt. productName ist raus, der Inhalt kommt dann auch in products rein.